browser/brave_wallet/solana_provider_browsertest.cc

```diff
@@ -1509,8 +1506,8 @@ IN_PROC_BROWSER_TEST_F(SolanaProviderTest, Request) {
   EXPECT_EQ(GetRequestResult(), kEncodedSignature);
 
   // signAllTransactions
-  CallSolanaRequest(base::StringPrintf(R"(
-    {method: "signAllTransactions", params: { message: ['%s', '%s'] }})",
+  CallSolanaRequest(content::JsReplace(R"(
+    {method: "signAllTransactions", params: { message: [$1, $2] }})",
```

Comment:
I'm ok with but curios what is the idea of changing some format strings to use positional specifiers?

Reply:
`JsReplace` is the testing helper for this type of replacement where we are inserting a JSON-like value into a JS/JSON string. Notice that I also removed the `'` there, because the representation is left to be however a JSON value is supposed to be represented.